tests/run_tests.py

```diff
@@ -21,4 +21,4 @@
     test_suite = unittest.defaultTestLoader.discover('tests', 'test*.py')
     test_runner = unittest.TextTestRunner()
     result = test_runner.run(test_suite)
-    sys.exit(not result.wasSuccessful())
+    sys.exit(len(result.errors) != 0 or not result.wasSuccessful())
```

Comment:
Do we now need a file after discovered a problem?

Reply:
No